src/peergos/server/Start.java

```diff
@@ -14,154 +14,186 @@
-                UserRepository userRepository = UserRepository.buildSqlLite(path, dht);
-                HttpCoreNodeServer.createAndStart(keyfile, passphrase, corenodePort, userRepository, userRepository, a);
-            } else {
-                int webPort = a.getInt("port", 8000);
-                URL coreAddress = new URI(a.getArg("corenodeURL", "http://localhost:" + HttpCoreNodeServer.PORT)).toURL();
-                String domain = a.getArg("domain", "localhost");
-                InetSocketAddress userAPIAddress = new InetSocketAddress(domain, webPort);
-
-                boolean useIPFS = a.getBoolean("useIPFS", true);
-                int dhtCacheEntries = 1000;
-                int maxValueSizeToCache = 50 * 1024;
-                ContentAddressedStorage dht = useIPFS ? new CachingStorage(new IpfsDHT(), dhtCacheEntries, maxValueSizeToCache) : RAMStorage.getSingleton();
-
-                // start the User Service
-                String hostname = a.getArg("domain", "localhost");
-
-                CoreNode core = HTTPCoreNode.getInstance(coreAddress);
-                MutablePointers mutable = HttpMutablePointers.getInstance(coreAddress);
-                MutablePointers pinner = new PinningMutablePointers(mutable, dht);
-
-                InetSocketAddress httpsMessengerAddress = new InetSocketAddress(hostname, userAPIAddress.getPort());
-                new UserService(httpsMessengerAddress, Logger.getLogger("IPFS"), dht, core, pinner, a);
-
-                if (a.hasArg("fuse")) {
-                    String username = a.getArg("username", "test01");
-                    String password = a.getArg("password", "test01");
-                    Path mount = Files.createTempDirectory("peergos");
-                    String mountPath = a.getArg("mountPoint", mount.toString());
-
-                    Path path = Paths.get(mountPath);
-                    path.toFile().mkdirs();
-
-                    System.out.println("\n\nPeergos mounted at " + path + "\n\n");
-
-                    NetworkAccess network = NetworkAccess.buildJava(webPort).get();
-                    Crypto crypto = Crypto.initJava();
-                    UserContext userContext = UserTests.ensureSignedUp(username, password, network, crypto);
-                    PeergosFS peergosFS = new PeergosFS(userContext);
-                    FuseProcess fuseProcess = new FuseProcess(peergosFS, path);
-
-                    Runtime.getRuntime().addShutdownHook(new Thread(() -> fuseProcess.close()));
-
-                    fuseProcess.start();
-                }
-            }
+            int webPort = a.getInt("port");
+            URL coreAddress = new URI(a.getArg("corenodeURL")).toURL();
+            String domain = a.getArg("domain");
+            InetSocketAddress userAPIAddress = new InetSocketAddress(domain, webPort);
+
+            boolean useIPFS = a.getBoolean("useIPFS");
+            int dhtCacheEntries = 1000;
```

Comment:
Should these values for``dhtCacheEntries`` and ``maxValueSizeToCache`` defaults that can be overridden via ``a``?